server/controllers/authController.js

```diff
@@ -25,7 +26,7 @@ exports.signup = async(request, response, next) => {
         });
 
         //Login the user immediately
-        return loggingInUsers(newUser._id, 201, response, 'Success', 'User has been Successfully Created');
+        return loginUserAfterSignup(newUser._id, 201, response, 'Success', 'User has been Successfully Created', newUser);
```

Comment:
can you move the status code to a config file? Looking at this I can't tell if this is a status code or not.

so maybe in your config file you can define an object 

```
status: {
'ok: 200,
'created: 201,
'not_found`: 404` etc..  
}
``` 

Then import it into your code

Reply:
These are just helper functions. In the underlying code which is currently stored in a separate file, it is very clear that they are status codes.

Reply:
I will make it clearer.

Reply:
Done